helper/consts/error.go

```diff
@@ -10,4 +10,8 @@ var (
 	// ErrStandby is returned if an operation is performed on a standby Vault.
 	// No operation is expected to succeed until active.
 	ErrStandby = errors.New("Vault is in standby mode")
+
+	// ErrVerIncompatible is returned if an operation is performed with a request
+	// coming from a core version is less than one stored in coreDataVersionPath
+	ErrVerIncompatible = errors.New("Vault data version is incompatible with client version")
```

Comment:
What's the purpose of this (I don't see it used in this PR). We don't really do client version checks -- was this meant to mean that the data version is incompatible with the _server_ version?